doc/apiv3-documentation.apib

```diff
@@ -3172,21 +3180,21 @@ the human readable name of custom fields.*
                         "title": "Version 1"
                     },
                     "availableWatchers": {
-                        "href": "/api/v3/work_packages/1528/available_watchers",
-                        "title": "Available Watchers"
+                        "href": "/api/v3/work_packages/1528/available_watchers"
                     },
                     "watch": {
                         "href": "/api/v3/work_packages/1528/watchers",
                         "method": "post",
-                        "data": {
-                            "user_id": 1
-                        },
-                        "title": "Watch work package"
+                        "payload": {
+                            "user": { "href": "/api/v3/users/1" }
+                        }
                     },
                     "addWatcher": {
```

Comment:
What about "removeWatcher"? Since the ticket does not explicitly state the necessity of either one I'd expected it for the sake of consistency. Also: Should both actions go to the same endpoint but use different http verbs (like POST and DELETE) or rather use separate endpoints?